build_wheel.sh

```diff
@@ -3,8 +3,25 @@
 THIS_SCRIPT=$0
 SCRIPT_DIR=$(dirname -- "$(readlink -f -- "$THIS_SCRIPT")")
 
+YEAR="${EBVERSIONGENTOO:-2017}"
+EXCLUDE_PYTHON_VERSIONS="/2\.\|/3.[5678]"
+OLDEST_SUPPORTED_NUMPY_VERSION=.2022a
+FLEXIBLAS_VERSION=3.0.4
+RAQM_VERSION=0.9.0
```

Comment:
It should be in its proper config file, not in the build script

Reply:
how about flexiblas? It's used by multiple config files.

Reply:
by using `flexiblas`, we are using the latest version returned by lmod

Reply:
the raqm module is also used in the pillow-simd config